realm/realm-library/src/main/cpp/io_realm_internal_Table.cpp

```diff
@@ -1235,15 +1305,17 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_Table_nativeFindFirstNull(JNIEnv*
                                                                          jlong columnIndex)
 {
     Table* pTable = TBL(nativeTablePtr);
-    if (!TBL_AND_COL_INDEX_VALID(env, pTable, columnIndex))
-        return jlong(-1);
-    if (!TBL_AND_COL_NULLABLE(env, pTable, columnIndex))
-        return jlong(-1);
+    if (!TBL_AND_COL_INDEX_VALID(env, pTable, columnIndex)) {
+        return to_jlong_or_not_found(-1);
```

Comment:
`to_jlong_or_not_found` is a confusing name to me...
what about remove the `to_jlong_or_not_found` and use:
`reinteprete_cast<jlong>(realm::not_found)` here?